test/Microsoft.AspNetCore.Server.IISIntegration.Tools.Tests/project.json

```diff
@@ -1,22 +1,18 @@
 {
   "dependencies": {
     "xunit": "2.1.0",
-    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*"
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-*"
   },
   "frameworks": {
-    "netcoreapp1.0": {
```

Comment:
I don't believe they can until this is merged: https://github.com/dotnet/cli/pull/2350
Also, the build we took today does not have this: https://github.com/NuGet/NuGet.Client/commit/9573f19ce94c84078e0d0e4f128bb24de7bfe140